requirements.txt

```diff
@@ -10,3 +10,4 @@ boto3==1.9.51
 networkx==2.1
 xlrd==1.1.0
 requests==2.21.0
+tabulate==0.8.3
```

Comment:
I'm assuming Pandas does have a way to print dataframes as nicely?

Reply:
I think not out of the box

Reply:
ok 👍